### ISSUE TITLE
any plan to add support for erb files?

### ISSUE THREAD

**Issue by joyoy96** [Shopify/vscode-shopify-ruby#597]
any plan to add support for erb files?

idk how you guys handle the erb files on vscode for the formatting and syntax highlighting

**Comment by vinistock** [comment]
Yes, here's the issue https://github.com/Shopify/ruby-lsp/issues/1055. There's currently no support for any features on ERB files other than basic syntax highlighting.

As outlined in the [README](https://github.com/Shopify/vscode-shopify-ruby?tab=readme-ov-file#vs-code-shopify-ruby), this extension doesn't provide any features, it's just an extension pack. So all feature issues can be found in the Ruby LSP repository.
